iolog.h

```diff
@@ -43,6 +43,7 @@ struct io_sample {
 	union io_sample_data data;
 	uint32_t __ddir;
 	uint64_t bs;
+	unsigned int priority_bit;
```

Comment:
This enlarges the IO samples (obviously), which adds to the memory pressure on tracking these. It also means that the fio client/server needs a revision jump, since we pass these on the wire. This is also why it's explicitly sized right now, so you should not use unsigned ints here.

Reply:
It looks to me like not all bits are used in the __ddir field. Would it make sense to avoid the additional memory footprint by rolling priority_bit into the existing uint32 with bit field declarations? I don't know if that would make the client/server updates easier or more complicated since I've never explored that code before.